pkg/component/component.go

```diff
@@ -526,17 +526,81 @@ func ValidateComponentCreateRequest(client *occlient.Client, componentSettings c
 //	componentConfig: Component configuration
 // Returns:
 //	err: Errors if any else nil
-func ApplyConfig(client *occlient.Client, componentConfig config.LocalConfigInfo, context string, stdout io.Writer) (err error) {
+func ApplyConfig(client *occlient.Client, componentConfig config.LocalConfigInfo, stdout io.Writer, cmpExist bool) (err error) {
```

Comment:
call cmpExist something similar to isUpdateDC ? to be more specific as to what it is actually used for?
can you please also update the comment of this function it seems not updated since a few prs before(including mine)

Reply:
added comments